blog/2015/10/python-debriefing-summary.html

```diff
@@ -0,0 +1,77 @@
+---
+layout: blog
+root: ../../..
+author: Alistair Walsh
+title: "A Summary of Debriefing Feedback on Our Python Lesson"
+date: 2015-10-08
+time: "01:00:00"
+category: ["Lessons", "Python"]
+---
+<!-- start excerpt -->
+<p>
+  Last month,
+  we discussed results from a survey of
+  <a href="{{page.root}}/blog/2015/09/thinking-about-teaching.html">how our instructors are teaching Python</a>.
+  We now have a summary of the feedback we've received in our bi-weekly debriefing meetings.
+  The recurring themes are:
+</p>
+<ol>
+  <li><p>A greater choice of exercises and multiple choice questions would allow instructors to select domain specific examples and cater to varying levels of learner.</p></li>
+  <li><p>Some instructors added an explanation of the IPython/Jupyter/SpyderIDE environment.</p></li>
+  <li><p>Some instructors added an explanation of basic Python datatypes before presenting the lessons.</p></li>
+  <li><p>Comments that there is too much material to fit into a workshop and that some sections seem rushed.</p></li>
+  <li><p>Request for a better explanation of the advantages of the Anaconda Distribution at the start of workshops and resources for post workshop learning.</p></li>
+  <li><p>Comments on presenters style were positive for funny and entertaining examples and negative for highly mathematical examples.</p></li>
+</ol>
+<!-- end excerpt -->
+<p>The details are listed below&mdash;we plan to use all of this when we redesign our basic Python lesson.</p>
+<ul>
+  <li>Departures
+    <ul>
+      <li><p>Changed lesson to be more relevant to audience +1</p></li>
+      <li><p>Instructor used own examples to change emphasis of lesson to loops</p></li>
+      <li><p>Re-organised the flow of lesson</p></li>
+    </ul>
+  </li>
+  <li>Additions
+    <ul>
+      <li><p>Explained differences between bash, python in the shell, IPython notebook</p></li>
```

Comment:
"IPython notebook" -> "IPython Notebook" since we use "Jupyter Notebook" in the next line.